embulk-output-jdbc/src/main/java/org/embulk/output/jdbc/setter/NStringColumnSetter.java

```diff
@@ -61,6 +61,6 @@ public void timestampValue(final Instant v) throws IOException, SQLException
     @Override
     public void jsonValue(Value v) throws IOException, SQLException
     {
-        defaultValue.setNString();
+        batch.setNString(v.toJson());
```

Comment:
Mainly for compatibility with the following changes.
https://github.com/embulk/embulk-output-jdbc/pull/320/files